.github/workflows/release-post-merge.yml

```diff
@@ -0,0 +1,73 @@
+# GitHub workflow for createing release.
+# Trigger release branch should be merge into main
+
+name: Create Release.
+
+on:
+  pull_request:
+    types: [closed]
```

Comment:
[q] hasn't the Andrey's suggestion worked out?

Reply:
I looked yesterday , but didn't find in the `push` GitHub event name of branch (because only for `release`something-something